packages/cysync-core/tsconfig.eslint.json

```diff
@@ -1,7 +1,10 @@
 {
   "compilerOptions": {
     "outDir": "dist",
-    "lib": ["ES2015"]
+    "lib": ["ES2015"],
+    "paths": {
+      "~/*": ["src/*"]
+    }
```

Comment:
I don't think we need this change as `tsconfig.json` already has `paths` defined and we are extending it here.